tikv-client/src/main/java/com/pingcap/tikv/JDBCClient.java

```diff
@@ -80,6 +74,23 @@ public String getPDAddress() throws SQLException {
     }
   }
 
+  public String getCurrentUser() throws SQLException {
```

Comment:
There are multiple methods sharing almost the same implementation. We can refactor it into one utilization function and reuse it in multiple places.

Reply:
ACK